src/Controller/AttributeInterface.php

```diff
@@ -0,0 +1,29 @@
+<?php
+/**
+ * SimpleMVC
+ *
+ * @link      http://github.com/simplemvc/framework
+ * @copyright Copyright (c) Enrico Zimuel (https://www.zimuel.it)
+ * @license   https://opensource.org/licenses/MIT MIT License
+ */
+declare(strict_types=1);
+
+namespace SimpleMVC\Controller;
+
+interface AttributeInterface
+{
+    /**
+     * Add an attribute for the next PSR-7 request in a pipeline
+     * 
+     * @param mixed $value
+     */
+    public function addRequestAttribute(string $name, $value): void;
```

Comment:
`mixed $value` as parameter (type missing)

Add @param string $name on the docblock

Reply:
We will add only phpdoc for the missing information, like `mixed` since we are supporting 7.4. The `$name` does not require a phpdoc since the type is already there. I also read your comment about supporting only PHP 8+ and I'm thinking about it.